### PR TITLE
Hardware location update

### PR DESCRIPTION
**Pull request by joachimweyl**
update row, pod, and cage for all of the hosts. Add the unitnumber
within the cage to infra and prod hosts. Will add test when we
get that information

### CODE REVIEW

**Review by mikthoma — APPROVED** [review]
Looks good!

**Comment by joachimweyl** [comment]
@larsks I used rebase to clean up my commits. Then had to force push to my repo. Is that what is causing the inability to merge? I did not change any files that had been changed by the 1 other merge you did on the public repo so there should not be any conflicts.

**Comment by larsks** [comment]
@joachimweyl you don't have merge privileges on this repository at the moment. I can update the permissions if you want, but we can change that if you'd like.

File: inventory/00-static/ocp-infra.yaml [hunk 1]
```diff
@@ -12,5 +12,17 @@ all:
                 vendor: lenovo
               hosts:
                 ctl-0.nerc-ocp-infra.rc.fas.harvard.edu:
+                  row: 07
+                  pod: C
+                  cage: 03
+                  unitnumber: 09-10
```
Comment (larsks) [inline inventory/00-static/ocp-infra.yaml]: I would prefer `unitnumber` to be a single value (typically, the lowest position the device occupies). If you want to record the device height, let's add a `height` parameter (but I'm not sure we need that at the moment; I guess I would be inclined to leave it out for now). Also, be careful writing numbers with leading zeroes: I don't think it's a problem here, but some tools interpret a leading 0 to mean "this is an octal number". If these are supposed to be numeric, strip the leading zero. If they are meant to be an opaque identifier, explicitly quote them (`cage: "03"`).

**Review by larsks — CHANGES_REQUESTED** [review]

File: inventory/00-static/ocp-infra.yaml (hunk 1 above)
Comment (joachimweyl) [reply inventory/00-static/ocp-infra.yaml]: Will do.

**Comment by joachimweyl** [comment]
> @joachimweyl you don't have merge privileges on this repository at the moment. I can update the permissions if you want, but we can change that if you'd like.

Got it, so it is not an issue with the PR, good. No need to change.

File: inventory/00-static/ocp-infra.yaml (hunk 1 above)
Comment (joachimweyl) [reply inventory/00-static/ocp-infra.yaml]: @larsks updated with no leading zero's and unit numbers only including the lower number.

**Comment by larsks** [comment]
@joachimweyl I've pushed a minor update that makes all the location metadata attributes of a top-level `location` variable, so:

```
              hosts:
                ctl-0.nerc-ocp-infra.rc.fas.harvard.edu:
                  location:
                    row: 7
                    pod: C
                    cage: 3
                    unit: 9
```

That makes it a little more convenient to use programatically. Other than that, looks good. If you're okay with this change I'll go ahead and merge it.

**Review by larsks — APPROVED** [review]
👍

**Pull request merged**